global.json

```diff
@@ -2,8 +2,5 @@
   "projects": [
     "src",
     "test"
-  ],
-  "sdk": {
-    "version": "1.0.0-preview2-1-003180"
```

Comment:
You want it back? Doesn't it completely bork KoreBuild's behavior of updating to our latest version?